shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/mysql/initdb.sql

```diff
@@ -18,8 +18,8 @@
 CREATE USER IF NOT EXISTS 'root'@'%' IDENTIFIED BY 'root';
 GRANT All privileges ON *.* TO 'root'@'%';
 
-CREATE DATABASE IF NOT EXISTS ds_0;
-CREATE DATABASE IF NOT EXISTS ds_1;
-CREATE DATABASE IF NOT EXISTS ds_2;
-CREATE DATABASE IF NOT EXISTS ds_3;
-CREATE DATABASE IF NOT EXISTS ds_4;
+CREATE DATABASE IF NOT EXISTS scaling_it_0;
+CREATE DATABASE IF NOT EXISTS scaling_it_1;
+CREATE DATABASE IF NOT EXISTS scaling_it_2;
+CREATE DATABASE IF NOT EXISTS scaling_it_3;
+CREATE DATABASE IF NOT EXISTS scaling_it_4;
```

Comment:
Do we need to drop existing database?

Reply:
the initdb.sql only use for docker mode, native mode not read the file